src/app/ApplicationTemplate.Tests/Business/PostServiceShould.cs

```diff
@@ -1,17 +1,15 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
+using AutoFixture.Xunit2;
 using FluentAssertions;
 using Xunit;
 
 namespace ApplicationTemplate.Tests
 {
-	public class PostServiceTests : TestBase<IPostService>
+	public class PostServiceShould : TestBase<IPostService>
```

Comment:
I like this change!

Reply:
Stolen from my Pluralsight lessons 👍 
I'll update on all other files in another pull request and add some documentation about the naming.